asset/css/bootstrap-vue.min.css

```diff
@@ -1 +1,3 @@
\ No newline at end of file
+/*!
```

Comment:
I think it'd be preferable to update the bootstrap-vue assets in a separate PR

Reply:
Ok! I'll revert the change.